### PR TITLE
Add GitHub action to create Quay tags for release

### PR DESCRIPTION
**Pull request by manaswinidas**
Github action to create Quay tags for ODH release

## Request review criteria:

Self checklist (all need to be checked):
- [ ] The developer has manually tested the changes and verified that the changes work
- [ ] Testing instructions have been added in the PR body (for PRs involving changes that are not immediately obvious).
- [ ] The developer has added tests or explained why testing cannot be added (unit or cypress tests for related changes)

If you have UI changes: 

- [ ] Included any necessary screenshots or gifs if it was a UI change.
- [ ] Included tags to the UX team if it was a UI/UX change.

After the PR is posted & before it merges:
- [ ] The developer has tested their solution on a cluster by using the image produced by the PR to `main`

### CODE REVIEW

File: .github/workflows/authorized-tag-creators.txt [hunk 1]
```diff
@@ -0,0 +1 @@
+manaswinidas
```
Comment (andrewballantyne) [inline .github/workflows/authorized-tag-creators.txt:1]: This is a list we should expand if it's dynamically read. 

Managers at a min (maybe a good start?), advisors should all be here (would make the most sense given their current responsibilities)... I don't have a problem with you being here, but I think if this is an actual grant list, we should definitely make sure it is prefilled with a good set of people so we are not locked out of anything.

**Review by andrewballantyne — CHANGES_REQUESTED** [review]
Have you tested this? If so, how'd you go about testing it?

File: .github/workflows/authorized-tag-creators.txt (hunk 1 above)
Comment (manaswinidas) [reply .github/workflows/authorized-tag-creators.txt:1]: I tested by pushing the changes to my fork and personal Quay repository with my secrets added to the repository. 

Ok, adding the advisors and managers. I didn't add you and Lucas coz you both already have access 😛

Comment (manaswinidas) [reply .github/workflows/authorized-tag-creators.txt:1]: Added myself coz I tested with the same.

Comment (andrewballantyne) [reply .github/workflows/authorized-tag-creators.txt:1]: Push the changes?

Comment (andrewballantyne) [reply .github/workflows/authorized-tag-creators.txt:1]: @manaswinidas can you take a screenshot of what you're seeing? I'm not seeing an update here. I think you're not pushing 🤔

Comment (manaswinidas) [reply .github/workflows/authorized-tag-creators.txt]: just added advisors and managers to the text file.. don't mind my previous comment I thought I wrote something weird

Comment (manaswinidas) [reply .github/workflows/authorized-tag-creators.txt]: About testing: here's a failed workflow run when I didn't add myself to the file: https://github.com/manaswinidas/odh-dashboard/actions/runs/12182131959/job/33980387355 and this is passing workflow run when I added myself to the file(this created a tag in my personal Quay repo on successful run): https://github.com/manaswinidas/odh-dashboard/actions/runs/12182092368/job/33980257364. Not sure whether you can access these links.

Comment (manaswinidas) [reply .github/workflows/authorized-tag-creators.txt]: And this is the workflow that I used there: https://github.com/manaswinidas/odh-dashboard/blob/main/.github/workflows/create-tag-release.yml

**Review by andrewballantyne — APPROVED** [review]
Lets see if this does what we are looking for. If this has issues we can just add more changes. It's not a product feature -- we can disable the github workflow if it goes wildly off the rails.

**Pull request merged**